### PR TITLE
[win32_event_log] Allow optional tagging by event_id

### PR DESCRIPTION
**Pull request by JohnLZeller**
Add config option for allowing the user to have events tagged by event_id of the form `event_id:<num>`.

### CODE REVIEW

**Comment by JohnLZeller** [comment]
Tested on Windows Server 2012 and it works beautifully.

**Comment by JohnLZeller** [comment]
@degemer since you last touched this check, would you mind giving me a quick r? :)

File: conf.d/win32_event_log.yaml.example [hunk 1]
```diff
@@ -1,4 +1,9 @@
 init_config:
+    # The (optional) tag_event_id setting will add an event id tag to each
+    # event sent from this check. Defaults to false.
+    # tag_event_id: true
+
+    tag_event_id: false
```
Comment (degemer) [inline conf.d/win32_event_log.yaml.example]: I think we kind of have an implicit convention that the default parameter should be commented, and nothing outside the comment, i.e. : 
```
 init_config:
    # The (optional) tag_event_id setting will add an event id tag to each
    # event sent from this check. Defaults to false.
    # tag_event_id: false
```

**Comment by JohnLZeller** [comment]
@degemer fixed! All good now?

File: checks.d/win32_event_log.py [hunk 2]
```diff
@@ -150,13 +151,14 @@ def _convert_event_types(self, types):
 
 
 class LogEvent(object):
-    def __init__(self, ev, api_key, hostname, tags, notify_list):
+    def __init__(self, ev, api_key, hostname, tags, notify_list, tag_event_id):
         self.event = ev
         self.api_key = api_key
         self.hostname = hostname
         self.tags = tags
```
Comment (degemer) [inline checks.d/win32_event_log.py]: I might be missing something, but why not use something like `self.tags = self._tags(tags, ev.EventCode) if tag_event_id else tags` ?
This way you don't need to have a `self.tag_event_id = tag_event_id`.

File: checks.d/win32_event_log.py [hunk 3]
```diff
@@ -190,6 +192,15 @@ def _wmi_to_ts(self, wmi_ts):
                       second=second, microsecond=microsecond) + tz_delta
         return int(calendar.timegm(dt.timetuple()))
 
+    def _tags(self, event, tags):
```
Comment (degemer) [inline checks.d/win32_event_log.py]: Why not just pass the `EventCode` as argument ? (since you don't use/need `event`)

**Comment by degemer** [comment]
Two last small comments, but LGTM otherwise !

**Comment by JohnLZeller** [comment]
Tested, and small changes incorporated :)

Merging

**Pull request merged**